catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/formulaeditor/FormulaEditorKeyboardTest.java

```diff
@@ -117,8 +155,8 @@ public void basicMathOperatorKeysTest() {
 	@Category({Cat.AppUi.class, Level.Smoke.class})
 	@Test
 	public void enterStringTest() {
-		onView(withId(R.id.brick_set_variable_edit_text))
-				.perform(click());
+		onBrickAtPosition(1).onChildView(withId(R.id.brick_change_size_by_edit_text)).perform(click());
+
```

Comment:
please uniformly use a coding style, in the above test it would have been fine to have the performs in one line, because its literally 10 times the same thing and theres not a lot of increased readability, but here it would still make sense...